### PR TITLE
Document Python profiler code provenance enablement

### PR DESCRIPTION
**Pull request by jd**
### What does this PR do?

Add a note about enabling code provenance on Python.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/profiler/enabling/python.md [hunk 1]
```diff
@@ -106,6 +106,13 @@ prof.start()
 
 You can configure the profiler using the [environment variables][5].
 
+### Code Provenance
```
Comment (maycmlee) [inline content/en/profiler/enabling/python.md:109]:
```suggestion
### Code provenance
```

File: content/en/profiler/enabling/python.md [hunk 2]
```diff
@@ -106,6 +106,13 @@ prof.start()
...
+
+The Python profiler supports code provenance reporting, which gives you access
```
Comment (maycmlee) [inline content/en/profiler/enabling/python.md:111]:
```suggestion
The Python profiler supports code provenance reporting, which gives you
```

File: content/en/profiler/enabling/python.md [hunk 3]
```diff
@@ -106,6 +106,13 @@ prof.start()
...
+to insight about library the running code is coming from. While this is
```
Comment (maycmlee) [inline content/en/profiler/enabling/python.md:112]:
```suggestion
insight into the library that is running the code. While this is
```

**Review by maycmlee — COMMENTED** [review]
Left some feedback! Let me know if it changed the meaning.

**Comment by jd** [comment]
@maycmlee could this be merged?

File: content/en/profiler/enabling/python.md [hunk 4]
```diff
@@ -106,6 +106,13 @@ prof.start()
 
 You can configure the profiler using the [environment variables][5].
 
+### Code provenance
+
+The Python profiler supports code provenance reporting, which gives you
```
Comment (maycmlee) [inline content/en/profiler/enabling/python.md:111]:
```suggestion
The Python profiler supports code provenance reporting, which provides
```

**Review by maycmlee — COMMENTED** [review]
Hey @jd! Just one more small edit.

**Review by maycmlee — APPROVED** [review]

**Pull request merged**